### PR TITLE
Make it more obvious that thing address must be contained in the serv…

### PR DESCRIPTION
**Pull request by boguslaw-wojcik**
…ice address.

### CODE REVIEW

File: adapter/thing_management.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Every adapter should define its own service name.
 
 | Field               | Type        | Example                        | Description                                                                                                                                                                              |
 |---------------------|-------------|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| address             | string      | `24`                           | An arbitrary unique identifier of the thing within the adapter, equal to node ID in Z-Wave and UDID in Zigbee.                                                                           |
+| address             | string      | `24`                           | An arbitrary unique identifier of the thing within the adapter. Must consist only from alphanumeric characters. For example in Z-Wave it is equal to Node ID, while in Zigbee to UDID.   |
```
Comment (BudzikM) [inline adapter/thing_management.md:110]: Use has to instead of must.

File: adapter/thing_management.md [hunk 2]
```diff
@@ -126,15 +126,15 @@ Every adapter should define its own service name.
...
+|--------------|-----------|--------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| name         | string    | `"out_bin_switch"`                               | Name of the service, as specified in [device services](/device_services/device_services.md) documentation.                                                                                                                        |
+| address      | string    | `"/rt:dev/rn:zw/ad:1/sv:out_bin_switch/ad:21_0"` | An address of the service starting from a resource type segment. Service address segment must contain thing address and optional group identification. See [topic format](/fimp/topic_format.md#topic-segments) for more details. |
```
Comment (BudzikM) [inline adapter/thing_management.md:132]: must -> shall for mandatory requirement

File: fimp/topic_format.md [hunk 3]
```diff
@@ -8,21 +8,22 @@ The segments are separated by a forward slash `/` which allows for selective sub
...
+| `pt`    | Parser type      | `j1`             | Possible values are: `j1` (JSON v1), `j1c1` (JSON v1 Compression Type 1 gzip).                      |
+| `mt`    | Message type     | `evt`            | Possible values are: `cmd` (command), `evt` (event), `rsp` (response).                              |
+| `rt`    | Resource type    | `dev`            | Possible values are defined in [resource types](#resource-types) section.                           |
+| `rn`    | Resource name    | `zw`             | An arbitrary name of the application or component, preferably snake_case.                           |
+| `ad`    | Resource address | `1`              | An arbitrary string defining address of the resource, resources on the hub will normally use `1`.   |
+| `sv`    | Service name     | `out_bin_switch` | Name of the device service.                                                                         |
+| `ad`    | Service address  | `3`, `3_1`       | A string defining address of the device service. Must follow `{THING_ADDRESS}[_{GROUP_ID}]` format. |
```
Comment (BudzikM) [inline fimp/topic_format.md:19]: In zigbee-ad I expect always %d_%d format -> provided resourceType is 'dev'
Sending '%d' possible on 'ad' resource type.

Comment (boguslaw-wojcik) [reply fimp/topic_format.md:19]: Yes it is.

File: adapter/thing_management.md (hunk 2 above)
Comment (boguslaw-wojcik) [reply adapter/thing_management.md:132]: Well, it's apple or oranges. There are two universally accepted standards I think:

https://www.ietf.org/rfc/rfc2119.txt

https://medium.com/@mrandig/iso-9001-2015-verbal-forms-dc50988a5aec

I believe it's just a matter of personal preference to choose one.

Comment (eternalamenity) [reply adapter/thing_management.md:132]: Zwave specification uses: https://www.ietf.org/rfc/rfc2119.txt
Just letting You know :D @boguslaw-wojcik

Comment (boguslaw-wojcik) [reply adapter/thing_management.md:132]: Guys, I personally I don't care. 

For me in documentation: 
* shall / have to / must = required
* should = recommended
* might / may / can = optional

I see no reason to limit vocabulary to strict ISO.

**Review by BudzikM — APPROVED** [review]

**Pull request merged**